app/api.py

```diff
@@ -124,6 +126,12 @@ async def authentication(form_data: Annotated[OAuth2PasswordRequestForm, Depends
     raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Server error")
 
 
+@app.get("/prices", response_model=List[schemas.PriceBase])
```

Comment:
@raphael0202 this is a simple v1, but I feel we'll need to have improvements like 
- using [fastapi-filters](https://github.com/arthurio/fastapi-filter) to have more complex filtering
- pagination & limit parameters
- should the endpoint answer a list, or a dict with keys like "data" ?

Reply:
pagination would be good to have indeed!
Better have a dict as a response, so we can add other fields if needed.
like `{"prices": [...]}`

Reply:
in Django Rest Framework the following format is returned
![image](https://github.com/openfoodfacts/open-prices/assets/7147385/704b0ed8-d403-4ff5-974c-231f23e61e30)

Reply:
I found [fastapi-pagination](https://github.com/uriyyo/fastapi-pagination), I'll add it in a follow-up PR